KlockanAPI/tests/KlockanAPI.Infraestructure.Tests/KlockanAPI.Infrastructure.Tests.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Upon review, I've found two files:

1. KlockanAPI.Infraestructure.Tests.csproj
2. KlockanAPI.Infrastructure.Tests.csproj

It appears that the first file, although incorrectly spelled, is the one already implemented. 
While we could refactor the names, this may introduce errors in related files such as the Dockerfile, so, we could retain the first file and delete the second one.

Reply:
This was a Visual Studio cache error on git GUI, I will delete the conflicting one.